pkg/issuer/acme/dns/route53/route53.go

```diff
@@ -83,17 +85,27 @@ func NewDNSProvider() (*DNSProvider, error) {
 // NewDNSProviderAccessKey returns a DNSProvider instance configured for the AWS
 // Route 53 service using static credentials from its parameters
 func NewDNSProviderAccessKey(accessKeyID, secretAccessKey, hostedZoneID, region string) (*DNSProvider, error) {
-
-	creds := credentials.NewStaticCredentials(accessKeyID, secretAccessKey, "")
-
 	r := customRetryer{}
 	r.NumMaxRetries = maxRetries
 
-	config := request.WithRetryer(aws.NewConfig(), r).WithCredentials(creds)
+	config := request.WithRetryer(aws.NewConfig(), r)
 
 	if region != "" {
 		config.WithRegion(region)
 	}
+
+	creds := credentials.NewChainCredentials([]credentials.Provider{
```

Comment:
I would prefer if the logic remained identical when an access key is specified, and if one isn't specified it uses the default credential chain rather than manually construct one including static credentials.

I think there are two good reasons to do this:
1. Simpler code (not having to construct a credential chain manually because the default can be used for non-static ones)
2. Better for non-aws users; with this change, someone not on aws who has an issue with their credentials may incur the cost of network traffic (the ec2 role provider being tried) before it fails